app/javascript/App/containers/WatchlistContainer/normalize.js

```diff
@@ -0,0 +1,12 @@
+import { normalize, schema } from 'normalizr'
```

Comment:
This looks cool, it's new to me, what does a normalizer do in JS?

Reply:
First time I'm using normalizr, but basically it's bad practice to nest objects in JSON (assuming you ever want to manipulate it), so what this does is move all nested objects to the top level with the object IDs as keys, and adds references to these keys where need be.